src/index.html

```diff
@@ -4,9 +4,390 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="catalog__header header">
+        <a href="#home" class="header__logo">
+          <img src="images/logo.png" alt="MOYO_logo">
+        </a>
+
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#" class="header__link header__link--is-active is-active">Apple</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Samsung</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#" class="header__link">Smartphones</a>
+            </li>
+
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+                data-qa="nav-hover"
+              >Laptops & Computers</a>
```

Comment:
```suggestion
              <a
                href="#"
                class="header__link"
                data-qa="nav-hover"
              >
                Laptops & Computers
              </a>
```

Reply:
14'th point in checklist

Reply:
checklist`s points finish at 13 point but I get what I need to do